website/source/docs/configuration/index.html.md

```diff
@@ -146,6 +146,11 @@ The following parameters are used on backends that support [high availability][h
   such as request forwarding are enabled. Setting this to true on one Vault node
   will disable these features _only when that node is the active node_.
 
+- `disable_performance_standby` `(bool: false)` - Specifies whether performance
```

Comment:
This is already in the Vault Enterprise Parameters below (with different text that should be trued up), if you want here instead for some reason please remove from below.